conf/application.conf

```diff
@@ -40,11 +40,28 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 
-# Custom error handler
-play.http.errorHandler = "config.ErrorHandler"
-
 play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
 
+#filters
+play.http.filters = play.api.http.EnabledFilters
+
+play.filters.enabled = [
+  play.filters.headers.SecurityHeadersFilter,
+  com.kenshoo.play.metrics.MetricsFilter,
+  uk.gov.hmrc.play.bootstrap.filters.frontend.HeadersFilter,
+  uk.gov.hmrc.play.bootstrap.filters.frontend.crypto.SessionCookieCryptoFilter,
+  uk.gov.hmrc.play.bootstrap.filters.frontend.deviceid.DeviceIdFilter,
+  uk.gov.hmrc.play.bootstrap.filters.LoggingFilter,
+  uk.gov.hmrc.play.bootstrap.filters.AuditFilter,
+  uk.gov.hmrc.play.bootstrap.filters.frontend.SessionTimeoutFilter,
+  play.filters.csrf.CSRFFilter,
+  uk.gov.hmrc.play.bootstrap.filters.CacheControlFilter,
+  uk.gov.hmrc.play.bootstrap.filters.MDCFilter
+]
+
+play.filters.enabled += filters.SessionIdFilter
```

Comment:
To my knowledge these don't need to be explicitly stated since in the application.conf we already have - 'play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"' which contains all this.

Reply:
We will check this one more time but SessionIdFilter was not working hence we added this

Reply:
SessionIdFilter is not working if remove the new added codes...

Reply:
I replied on slack but I think custom filters are meant to be added in a different way that doesn't require re-introducing all those other filters a second time - https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=PLATOPS&title=How+to+remove+Play+GlobalSettings